packages/driver/test/testUtil.ts

```diff
@@ -90,6 +90,10 @@ export const getServerCommand = (
     srvcmd = process.env.EDGEDB_SERVER_BIN;
   }
 
+  if (!(process.env.EDGEDB_SERVER_BIN || process.env.GITHUB_ACTIONS)) {
```

Comment:
```suggestion
  if (!(process.env.EDGEDB_SERVER_BIN || process.env.CI)) {
```
I think `CI` is the more commonly used env var?